### PR TITLE
Helm fixes

### PR DESCRIPTION
**Pull request by j4ys0n**
Update helm configmap to properly transform network.yaml > channels.peers & fix typo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,8 +88,17 @@ We also include a helm chart for Kubernetes deployments. First set your `values.
 
     secrets:
         peer:
-            cert: hlf-peer--peer0-cert
-            key: hlf-peer--peer0-key
+        cert: hlf--peer-admincert
+        # itemKey can be defined if there is a secret with multiple items stored inside.
+        certItem: cert.pem
```
Comment (atoulme) [inline README.md:93]: looking at your helm chart, I think the values are not indented properly. I might be wrong, as it's hard to eyeball this type of whitespace changes. Please lmk.

File: helm-chart/fabric-logger/values.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ replicaCount: 1
 
 image:
   repository: ghcr.io/splunkdlt/fabric-logger
-  version: 4.2.0
+  version: 4.2.1
```
Comment (atoulme) [inline helm-chart/fabric-logger/values.yaml:9]: Are those values updated automatically on release?

Comment (j4ys0n) [reply helm-chart/fabric-logger/values.yaml:9]: good question - i looked at the actions and didn't see anything that suggested the versioning was automatic.

Comment (ccordi) [reply helm-chart/fabric-logger/values.yaml:9]: New releases should be triggered on push to master for feat and fix https://github.com/splunk/fabric-logger/blob/master/.github/workflows/ci.yaml#L101

Comment (j4ys0n) [reply helm-chart/fabric-logger/values.yaml:9]: ah, got it - i'll revert the version change

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "fabric-logger",
-  "version": "4.2.0",
+  "version": "4.2.1",
```
Comment (ziegfried) [inline package.json:3]: Please don't manually "bump" the version. We're using semantic-release in this repo.

Comment (j4ys0n) [reply package.json:3]: yep - got it. didn't realize that. just talked with Chris about it.

**Review by ccordi — APPROVED** [review]
LGTM.

**Pull request merged**